source/fundamentals/atlas-search.txt

```diff
@@ -0,0 +1,606 @@
+.. _csharp-atlas-search:
+
+============
+Atlas Search
+============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide you can learn how to use the ``Search`` class to build a ``$search``
+aggregation pipeline stage with the {+driver-long+}.
+
+To learn more about the ``$search`` pipeline stage, see :manual:`$search
+</reference/operator/aggregation/search/>`.
+
+.. note:: Only Available on Atlas for MongoDB v4.2 and Later
+
+   The ``$search`` aggregation-pipeline operator is available only for collections hosted
+   on :atlas:`MongoDB Atlas </>` clusters running MongoDB v4.2 or later that are
+   covered by an :atlas:`Atlas search index </reference/atlas-search/index-definitions/>`.
+   Learn more about the required setup and the functionality of this operator
+   from the :ref:`Atlas Search <fts-top-ref>` documentation.
+
+The examples in this guide use the following documents in a collection called
+``guitars``:
+
+.. code-block:: json
+ 
+   { "_id": 1, "make": "Fender", "description": "Classic guitars known for their versatility.", "establishedYear": 1946, "in-stock": true, "rating": 9 }
+   { "_id": 2, "make": "Gibson", "description": "Classic guitars known for their rich, full tones.", "establishedYear": 1902, "in-stock": true, "rating": 8 }
+   { "_id": 3, "make": "PRS", "description": "High-end guitars known for their quality.", "establishedYear": 1985, "in-stock": true, "rating": 9 }
+   { "_id": 4, "make": "Kiesel", "description": "Quality guitars made only for custom orders.", "establishedYear": 2015, "in-stock": false }
+   { "_id": 5, "make": "Ibanez", "description": "Well-crafted guitars used by many professional guitarists.", "establishedYear": 1957, "in-stock": true, "rating": 7 }
+   { "_id": 6, "make": "Strandberg", "description": "Modern guitars known for their headless models.", "establishedYear": 1982, "in-stock": false }
+
+The following ``Guitar`` class models the documents in this collection.
+
+.. code-block:: csharp
+
+   public class Guitar
+   {
+       public int Id { get; set; }
+       public string Make { get; set; }
+       public List<string> Models { get; set; }
+       public int EstablishedYear { get; set; }
+       [BsonElement("in-stock")]
+       public bool InStock { get; set; }
+       public int? Rating { get; set; }
```

Comment:
I chose to not include location from the start since I feel like it clutters up the documents unnecessarily, and it's only used for two examples out of many. Including it from the start would mean that every example output would include it as well, so I wanted to try and keep that a little cleaner. 
I was a bit torn on whether or not I should do that though, so happy to take input on if you agree or not

Reply:
I think it depends on whether or not you think there will be more. For one, I completely agree with your logic. For two, I think it works. If it was more than two, I would say it should be changed.

Reply:
Side note - I'm fine resolving this thread, but left it open to make sure you saw it.

Reply:
I don't think there will be more anytime in the near future. Definitely agree though if we do add any more examples we should just change the whole thing